glean-core/ios/Glean/Dispatchers.swift

```diff
@@ -20,21 +20,27 @@ class Dispatchers {
 
     private let logger = Logger(tag: Constants.logTag)
 
-    // This is the task queue that all Glean background operations will be executed on.  It is currently
-    // set to be a serial queue by setting the `maxConcurrentOperationsCount` to 1, but could allow for
-    // concurrent operations by increasing the parameter.
-    lazy var operationQueue: OperationQueue = {
+    // This is a task queue for all Glean background operations that are required to be executed in order.
+    // It is currently set to be a serial queue by setting the `maxConcurrentOperationsCount` to 1
+    lazy var serialOperationQueue: OperationQueue = {
         var queue = OperationQueue()
-        queue.name = "Glean dispatch queue"
+        queue.name = "Glean serial dispatch queue"
         queue.maxConcurrentOperationCount = 1
         return queue
     }()
 
+    // This is a task queue for all Glean background operations that need be executed concurrently.
```

Comment:
Actually, we don't guarantee concurrency, it's just a possibility.

```suggestion
    // This is a task queue for all Glean background operations that can be executed concurrently.
```